management_api.go

```diff
@@ -156,6 +159,33 @@ func (e *Enforcer) RemovePolicy(params ...interface{}) (bool, error) {
 	return e.RemoveNamedPolicy("p", params...)
 }
 
+// UpdatePolicy updates an authorization rule from the current policy.
+func(e *Enforcer) UpdatePolicy(p1 string,p2 string) (bool, error){
```

Comment:
p1, p2 should be []string.

Reply:
If use []string, we must use UpdatePolicy by `e.UpdatePolicy([]string{xxx,xxx,xxx}, []string{xxx,xxx,xxx}`.
I think this way is not fine....

Reply:
@hsluoyz what do you think?

Reply:
I think `e.UpdatePolicy([]string{xxx,xxx,xxx}, []string{xxx,xxx,xxx}` is OK for me, if we don't have better solutions.